marklogic-data-hub-central/ui/src/pages/Load.test.tsx

```diff
@@ -38,38 +40,40 @@ describe('Load component', () => {
         // test 'Add New' button
         expect(queryByText('Add New')).not.toBeInTheDocument();
 
-        //test settings
-        fireEvent.click(getByTestId('testLoad-settings'));
-        expect(await(waitForElement(() => getByText('Target Database:')))).toBeInTheDocument();
-        expect(getAllByText('Save')[0]).toBeDisabled();
-        fireEvent.click(getAllByText('Cancel')[0]);
+        // test settings
+        await act(async () => {
+            await fireEvent.click(getByTestId('testLoad-settings'));
+        });
+        expect(await(waitForElement(() => getByText('Target Database')))).toBeInTheDocument();
+
+        expect(getByText('Save')).toBeDisabled();
+        await act(async () => {
+            await fireEvent.click(getByText('Cancel'));
+        });
+        // test delete
+        expect(queryByTestId('testLoad-delete')).not.toBeInTheDocument();
 
         //test edit
         fireEvent.click(getAllByText('testLoad')[0]);
-        expect(await(waitForElement(() => getByText('Edit Data Load')))).toBeInTheDocument();
+        expect(getByText('Edit Data Load')).toBeInTheDocument();
         expect(getAllByText('Save')[0]).toBeDisabled();
         fireEvent.click(getAllByText('Cancel')[0]);
 
-        // test delete
-        expect(queryByTestId('testLoad-delete')).not.toBeInTheDocument();
-
         // Check card layout
         fireEvent.click(getByLabelText('switch-view-card'));
 
         // test 'Add New' button
         expect(queryByText('Add New')).not.toBeInTheDocument();
 
         // test settings
-        fireEvent.click(getByLabelText('icon: setting'));
-        expect(await(waitForElement(() => getByText('Target Database:')))).toBeInTheDocument();
-        expect(getAllByText('Save')[0]).toBeDisabled();
-        fireEvent.click(getAllByText('Cancel')[0]);
-
-        //test edit
-        fireEvent.click(getByTestId('testLoad-edit'));
-        expect(await(waitForElement(() => getByText('Edit Data Load')))).toBeInTheDocument();
-        expect(getAllByText('Save')[0]).toBeDisabled();
-        fireEvent.click(getAllByText('Cancel')[0]);
+        await act(async () => {
+            await fireEvent.click(getByLabelText('icon: setting'));
+        });
+        expect(await(waitForElement(() => getByText('Target Database')))).toBeInTheDocument();
+        expect(getByText('Save')).toBeDisabled();
+        await act(async () => {
+            await fireEvent.click(getByText('Cancel'));
+        });
```

Comment:
tests for edit seems to have been removed.

Reply:
Do you mean this?

https://github.com/marklogic/marklogic-data-hub/pull/4058/files#diff-96cd2afde423c81c4d73a3bd147090c4R56

Reply:
I am viewing the diffs in split view. Maybe that will help.
<img width="259" alt="image" src="https://user-images.githubusercontent.com/16124406/84190374-aeca8b00-aa4b-11ea-8b8f-ae0dcee6a8e0.png">

<img width="1002" alt="image" src="https://user-images.githubusercontent.com/16124406/84190458-d4f02b00-aa4b-11ea-9127-9bcf1063266b.png">